src/pubsub/mod_pubsub_db_rdbms_sql.erl

```diff
@@ -690,3 +669,9 @@ esc_string(String) ->
 esc_int(Int) ->
     mongoose_rdbms:use_escaped_integer(mongoose_rdbms:escape_integer(Int)).
 
+convert_sql_nidx({selected, [{Nidx}]}) ->
```

Comment:
I just realize that it may be helpful for someone new to this part of the code if there is a comment somewhere in the code which DB returns what. So that it's easier to understand the code. You and me understand it (at least today). I'm not sure how easy would it be to understand for anyone who didn't work on this change.

Reply:
I've marked which clause applies to which DB. I think that's enough to understand :)